management/dns_update.py

```diff
@@ -303,6 +303,23 @@ def has_rec(qname, rtype, prefix=None):
 		if not has_rec(qname, rtype):
 			records.append((qname, rtype, value, explanation))
 
+	# Adds autoconfiguration A records for all domains.
+	# mta-sts.* - required A record for mta-sts (serving the policy)
+	mta_sts_records = [
+		("mta-sts", "A", env["PUBLIC_IP"], "Provides MTA-STS support"),
+		("mta-sts", "AAAA", env["PUBLIC_IPV6"], "Provides MTA-STS support"),
```

Comment:
I think this needs: `if env.get('PUBLIC_IPV6')`

Reply:
I copied the line from the auto discovery line above

`
("autoconfig",` "AAAA", env["PUBLIC_IPV6"], "Provides email configuration autodiscovery support for Thunderbird Autoconfig.")
`
It works at my site; i'll try it with your proposal, but in that case, the lines for autoconfig and auto discovery also needs changing.

Thanks for your fast response.

Reply:
I double checked it's removed in the iteration below. So it was fine :)

Reply:
It seems to work both, i changed it, before your reply.